update_translations.py

```diff
@@ -0,0 +1,116 @@
+import subprocess
```

Comment:
Ah, I hadn't seen the other files in this repo before. Still think it's worth doing, but doesn't need to block this PR.